AzureMS/src/server/LifeEntity/MobEntity/MonsterEntity/MapleMonster.java

```diff
@@ -1196,7 +1196,7 @@ public final void handleSteal(MapleCharacter chr) {
                         idrop = new Item(d.itemId, (byte) 0,
                                 (short) (d.Maximum != 1 ? Randomizer.nextInt(d.Maximum - d.Minimum) + d.Minimum : 1),
                                 (byte) 0);
-                        idrop.setGMLog(chr.getName() + "가 스틸로 인한 아이템 훔치기로 얻은 아이템");
+                        idrop.setGMLog(chr.getName() + "obtained items using Steal.");
```

Comment:
chr.getName() + " obtained items using Steal."

Reply:
hey. I just followed the original format, kay 🤙